installation/install.sh

```
@@ -2574,6 +2583,238 @@ install_morph_server(){
+    if [ -z "$installer_bin" ]; then
+        fail "license server installer package for $BASEARCH not found"
+    fi
+
+    echo_title "Install License Server"
+    echo ""
+    trap 'traplogger $LINENO "$BASH_COMMAND" $?'  DEBUG
+
+    prepare_license_server_password
+    show_spinner prepare_license_server_user_and_db
+
+    LICENSE_SERVER_INSTALL_OUTPUT=`mktemp`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)